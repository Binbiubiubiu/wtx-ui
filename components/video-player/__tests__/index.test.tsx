import React from 'react';
import { render } from 'enzyme';
import VideoPlayer from '..';
import toJson from 'enzyme-to-json';

describe('VideoPlayer', () => {
  it('should render VideoPlayer components', () => {
    const wrapper = render(
      <>
        <VideoPlayer src="http://meigui.qqqq-kuyun.com/20190406/976_420fc19b/index.m3u8" />
      </>,
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('should style output is not null', () => {
    expect(require('../style')).not.toBeNull();
  });
});
