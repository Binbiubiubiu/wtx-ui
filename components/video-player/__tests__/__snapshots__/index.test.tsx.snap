// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer should render VideoPlayer components 1`] = `
<div
  class="wtx-video-player__wrapper"
  style="width:400px;height:240px"
>
  <video
    class="video-js wtx-video-player"
  />
</div>
`;
